front/app/components/HookForm/QuillMultilocWithLocaleSwitcher/index.tsx

```diff
@@ -57,7 +57,7 @@ const QuillMultilocWithLocaleSwitcher = ({ name, ...rest }: Props) => {
           <QuillMultilocWithLocaleSwitcherComponent
             {...field}
             {...rest}
-            id={name}
+            id={name.includes('.') ? 'quill-multiloc-editor' : name}
             valueMultiloc={{ ...defaultValue, ...field.value }}
```

Comment:
Uses a default ID if the name doesn't work as a valid ID.

Reply:
Any chance we could replace the `.` with `_` in those cases?

Reply:
From what I understand those names are generated by JSON Forms - so I didn't find a way to do so. I can try taking a second look though.

Reply:
Are the names descriptive? From what I can see, we are trying to avoid `.` in the id. We can still maintain the names by doing some thing like this.

![image](https://user-images.githubusercontent.com/12659000/201872409-14929637-6b02-4ae1-9309-c9ba5b3c378b.png)